staging/vhost-device-gpu/src/main.rs

```diff
@@ -34,15 +34,34 @@ struct GpuArgs {
     /// vhost-user Unix domain socket.
     #[clap(short, long, value_name = "SOCKET")]
     socket_path: PathBuf,
+    /// Select the GPU mode: either 'gfxstream' or 'virglrenderer'
+    #[clap(short, long, value_enum)]
+    renderer: RenderMode,
```

Comment:
The description mentioning possible values is redundant here. Clap already generates the possible values. This is the output of `--help`

Usage: vhost-device-gpu --socket-path <SOCKET> --renderer <RENDERER>
```
A virtio-gpu device using the vhost-user protocol.

Usage: vhost-device-gpu --socket-path <SOCKET> --renderer <RENDERER>

Options:
  -s, --socket-path <SOCKET>  vhost-user Unix domain socket
  -r, --renderer <RENDERER>   Select the GPU mode: either 'gfxstream' or 'virglrenderer' [possible values: gfxstream, virglrenderer]
  -h, --help                  Print help
  -V, --version               Print version
```